packages/ag-charts-community/project.json

```diff
@@ -15,7 +15,7 @@
         "rollupConfig": "packages/ag-charts-community/rollup.config.cjs",
         "project": "packages/ag-charts-community/package.json",
         "compiler": "tsc",
-        "format": ["cjs"]
+        "format": ["cjs", "esm"]
```

Comment:
I'm a bit concenred about the performance impact here - is it doubling our reload times?

Reply:
I don't think it's that bad. I'm also not sure if we have any other options.